### DISCUSSION TITLE
[Feature] Enable Mobile app to connect with client certificate

### DISCUSSION THREAD

**Discussion by maratsh** [immich-app/immich#4363] [Feature Request]
### The feature

As user, I might want to additionally protect my internet-facing immich instance from possible attacks on auth pass-trough vulnerabilities. 

Mobile must have optional client certificate selection button. User should be able to use this button to select client certificate. Mobile client then will use this certificate to pass certificate auth. After establishing normal https connection  normal user/password or OIDC auth can be used.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1611
